api/facades/admin/AdminV3.ts

```diff
@@ -63,6 +63,7 @@ export interface LoginRequest {
   credentials: string;
   macaroons: Macaroon[][];
   nonce: string;
+  token?: string;
```

Comment:
There are changes to some of the facades if the schema.json changed anything between versions.